### PR TITLE
finished changes on spreadsheet

### PR DESCRIPTION
**Pull request by ihateerrors**
A decent amount of changes with fonts, styles and added a lot of content as well. Changed layout of the site.  Header needs a bit of a work, took too much time figuring out best layout, need a second eye.

### CODE REVIEW

File: css/base.css [hunk 1]
```diff
@@ -506,6 +509,21 @@ form [type="submit"] {
   font-weight: 300;
   font-style: normal;
 }
+@font-face {
+  font-family: 'Bahnschrift';
+  src: url('path/to/fonts/Bahnschrift-Regular.woff2') format('woff2'),
```
Comment (benbogard) [inline css/base.css:514]: path to... where?

File: css/base.css [hunk 2]
```diff
@@ -265,6 +265,9 @@ h4 {
   font-size: 1.7rem;
   font-weight: 700;
 }
+h5 {
+  font-style: normal !important;
```
Comment (benbogard) [inline css/base.css:269]: can we push this style further down the cascade so that !important is not needed?

File: south-pine-street.html [hunk 3]
```diff
@@ -541,21 +541,21 @@ <h2 class="footer-header" id="contact-us">
...
-                <br /><strong>Web:</strong>
+                <br />
                 <a
                   href=" https://www.soundtransit.org/system-expansion/south-tacoma-station-access-improvements"
                   target="_blank"
                   >South Tacoma</a
                 >,
                 <a
-                  href="https://www.soundtransit.org/system-expansion/lakewood-station-access-improvements"
+                  href="https://www.soundtransit.org/system-expansion/lakewood-station-access-improvements "
```
Comment (benbogard) [inline south-pine-street.html:558]: extra space on the end of the url

**Review by benbogard — CHANGES_REQUESTED** [review]
@Gengur123 added some comments/questions. Reckon we should fold in the actual fonts into the codebase within this PR. Also, it is !important to do our best to not use !important ;)

File: css/base.css (hunk 1 above)
Comment (ihateerrors) [reply css/base.css]: Fonts weights were kinda strange, was trying to find a way around this but this definitely wasn't it hahaha

**Review by benbogard — APPROVED** [review]

**Pull request merged**